src/products/views/ProductList/filters.ts

```diff
@@ -207,15 +208,18 @@ export function getFilterVariables(
         }))
       : null,
     categories: params.categories !== undefined ? params.categories : null,
+    channel: channel || null,
     collections: params.collections !== undefined ? params.collections : null,
     isPublished:
       params.status !== undefined
         ? params.status === ProductStatus.PUBLISHED
         : null,
-    price: getGteLteVariables({
-      gte: parseFloat(params.priceFrom),
-      lte: parseFloat(params.priceTo)
-    }),
+    price: channel
```

Comment:
Is there a situation where channel is not chosen?
Don't have staging to check it 🤷‍♂️

Reply:
it's for situation when channel is "loading"